### DISCUSSION TITLE
Language selector frozen after v8 installation

### DISCUSSION THREAD

**Discussion by alffonsse** [elementary/triage#631] [Q&A]
Hi, I just tried to install version 8 on my laptop. The installation completed successfully buf after the initial reboot I am stuck on the out of the box language selector. It does not respond to any input (mouse or keyboard) so I cannot continue with setting up the fresh install.
Strangely the menus at the top right (sound, network...) are responding to the mouse input.
How can I proceed to continue with the setup?

**Comment by teamcons** [comment]
Do Ctrl+Alt+Backspace work ?

I dont remember, this language selector shows up before any desktop is loaded, right ? 

What graphic card do you use ? Usually weird strange things comes from there

**Reply by alffonsse** [reply]
So I actually tried reinstalling multiple times juste before I saw your response. I haven't tried Ctrl+Alt+Backspace.
> I dont remember, this language selector shows up before any desktop is loaded, right ?

Yes.

It seems that if I click anywhere just after the language selector pops up it puts it in a weird state where it doesn't take any input anymore.
However, if I wait a few minutes, everything works fine so I was abel to complete the setup and access my user session.
My PC is using the integrated Intel graphics but I also have an NVIDIA Quadro M1000 in there.

**Reply by teamcons** [reply]
I am sorry you get this issue... But i am relieved somehow you could complete. This sound weird.
This annoys me, i cant pinpoint what causes this, and because no session is created yet, theres no running DE and root is disabled im not sure of ways to "hotfix" this
Maybe theres something in the background struggling with NVIDIA? Nvidia stuff on linux is pain...

**Comment by SystemError12** [comment]
I had issues with the language selection as well. What I tried (and didn't work) was to set English as the main language and combine it with a different language's keyboard layout. I tried several time to install it with that option...to no avail. 

I ended up installing English + English keyboard layout just to change it later after the installation process, which just requires a few mouseclicks inside the logged in and running os. 

I am not sure if it's related but might be worth a try. Good luck ;)
